Revert from hwe kernel to normal lts kernel

jpalermo opened the pull request:
We've seen OOM problems when running cgroups v1 with the 6.5 kernel line.

Resolves #318

The plan is to test this before releasing it. So we'd like to merge it and then use the test stemcells to verify hardware compatibility with the kernel.

Once the stemcells are built in the pipeline, we'll reference the URLs in #318 so people have a chance to take a look at them.

Earliest date we'd want to release this is Monday March 11th.

Commit:
Revert from hwe kernel to normal lts kernel

We've seen OOM problems when running cgroups v1 with the 6.5 kernel line.

Resolves [#318]

lnguyen reviewed (dismissed):
:+1: LGTM

ramonskie commented:
you need to adjust https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-jammy/master/stemcell_builder/stages/static_libraries_config/apply.sh#L16 to 5.15
and this probably needs a check as well https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/ubuntu-jammy/master/stemcell_builder/lib/prelude_apply.bash#L50

Commit:
Fixes to tests and kernel version

Commit:
fix copy error

cunnie committed:
The path for libsubcmd has changed between kernel versions, add the alternate path

8fecc2921c27be30f4558378c9804c618c6ebc36 originally changed this, now
we are adding both for compatibility.

Signed-off-by: Joseph Palermo <joseph.palermo@broadcom.com>
Co-authored-by: Joseph Palermo <joseph.palermo@broadcom.com>
